src/v3/src/components/Widget/style.css

```diff
@@ -179,3 +179,32 @@ span.strong {
 span.no-translate {
   white-space: nowrap;
 }
+
+.app-store-logo {
+  display: block;
+  /* stylelint-disable-next-line liberty/use-logical-spec */
+  margin: 10px auto;
+  /* stylelint-disable-next-line liberty/use-logical-spec */
+  height: 50px;
+  /* stylelint-disable-next-line liberty/use-logical-spec */
+  width: 320px;
+  background-repeat: no-repeat;
+  background-position: center center;
+}
+
+.android-app-store-logo {
+  background-image: url('../../../../../assets/img/appstore/google-play-store.svg');
+}
+
+.ios-app-store-logo {
+  background-image: url('../../../../../assets/img/appstore/apple-app-store.svg');
+  transform: scale(1.4);
+}
+
+.windows-app-store-logo {
+  background-image: url('../../../../../assets/img/appstore/windows-app-store.svg');
+}
+
+.osx-app-store-logo {
+  background-image: url('../../../../../assets/img/appstore/osx-app-store.svg');
```

Comment:
The best solution would be to create a new component in `src/v3/src/components` called something like `ImageLink` or `LinkImage` that wraps an image in an anchor `a` tag. We have a good example where we do this in `ConsentHeader`, [see here.](https://github.com/okta/okta-signin-widget/blob/b3520b4b3c51e7e19deefd547fa8ec7b6474d715/src/v3/src/components/ConsentHeader/ConsentHeader.tsx#L45-L83)

The code I linked ^ is from our `odyssey-1.x` branch and it uses a [new `Image` functional component](https://github.com/okta/okta-signin-widget/blob/odyssey-1.x/src/v3/src/components/Image/Image.tsx) that we created in that branch and will be merged into `master` in the next few days. I would suggest copying over the code for this `Image` component and using it when you create your new `ImageLink` component. One thing to keep in mind is that the `ImageLink` component will have to be a `UiSchemaElementComponent` so that it can be rendered in `transformOktaVerifyEnroll`, so the function signature will be something like:

```
const ImageLink: UISchemaElementComponent<{
  uischema: ImageLinkElement
}> = ({ uischema }) => {
INSERT COMPONENT CODE HERE...
}
```

Reply:
@trevoring-okta please see https://github.com/okta/okta-signin-widget/pull/3526/commits/3aea04a4eacab16efa4a6af83b1b4896160fa273